dwl.c

```diff
@@ -888,6 +888,9 @@ createnotify(struct wl_listener *listener, void *data)
 	c->surface.xdg = xdg_surface;
 	c->bw = borderpx;
 
+	/* set monitor, then it will be overwritten by applyrules */
+	setmon(c,selmon, 0);
```

Comment:
I think calling `applyrules` here is better than `setmon`, but it can't get alacritty's appid so I have to call setmon and modify it to assign tags even if it is the same monitor